src/preact-render-spy.js

```diff
@@ -164,6 +164,12 @@ const verifyFoundNodes = wrapper => {
   }
 };
 
+const verifyOnlySingleNode = wrapper => {
+  if (wrapper.length !== 1) {
+    throw new Error('preact-render-spy: component method can only be used on a single node');
```

Comment:
```js
const verifyOnlySingleNode = (wrapper, methodName) => {
```

```js
    throw new Error(`preact-render-spy: ${methodName} method can only be used on a single node`);
```

Reply:
Waiting for #69 to be merged.

Reply:
Is there anything else blocking this PR from being merged?